resources/kcp/charts/kyma-environment-broker/values.yaml

```diff
@@ -2,7 +2,7 @@ global:
   images:
     kyma_environment_broker:
       dir:
-      version: "PR-2546"
+      version: "PR-2555"
```

Comment:
nit: given it's change exclusively in the tests, there is no need to bump the image (but it doesn't hurt either, `pull-keb-build` pipeline produced the image)